packages/test/src/test-integration-workflows.ts

```diff
@@ -1303,3 +1311,45 @@ test('Count workflow executions', async (t) => {
     ],
   });
 });
+
+test('can register search attributes to dev server', async (t) => {
+  const { startWorkflow } = helpers(t);
+  const key = defineSearchAttributeKey('new-search-attr', SearchAttributeType.INT);
+  const newSearchAttribute: SearchAttributePair = { key, value: 12 };
+
+  // Ensure that search attribute is not registered on current env.
```

Comment:
I don't think checking that the SA isn't already registered is pertinent. The actual test (below) creates a new/distinct test server anyway. If you're really worried about this, you can append a random UUID to the SA's name, but even that seems overkill.

Reply:
Not sure why I did that given I'm creating a new test env right below... removed.